crates/iota-data-ingestion-core/src/history/reader.rs

```diff
@@ -77,11 +77,12 @@ impl HistoricalReader {
 
         files.sort_by_key(|f| f.checkpoint_seq_range.start);
 
-        assert!(
-            files
-                .windows(2)
-                .all(|w| w[1].checkpoint_seq_range.start == w[0].checkpoint_seq_range.end)
-        );
+        // TODO this is panicking for the devnet ingestion/historical/MANIFEST
```

Comment:
Why? And what do we do about it?

Reply:
There is already an issue I got pointed to: https://github.com/iotaledger/devops/issues/494

I'll uncomment those lines and remove the TODO, because there's nothing that needs to change in the code but rather on the devops side of things.